src/trackedit/internal/au3/au3interaction.cpp

```diff
@@ -317,6 +317,67 @@ void Au3Interaction::trimOrDeleteOverlapping(WaveTrack* waveTrack, secs_t begin,
     }
 }
 
+std::optional<secs_t> Au3Interaction::getShortestClipDuration(const ClipKeyList &clipKeys)
```

Comment:
according to the current style, it is better to simply call it: `shortestClipDuration`

This method can be const